assembl/graphql/post.py

```diff
@@ -330,13 +357,16 @@ def mutate(root, args, context, info):
                         subject_langstring = models.LangString.create(
                             new_subject, locale)
 
+            now = datetime.utcnow()
             new_post = cls(
                 discussion=discussion,
                 subject=subject_langstring,
                 body=body_langstring,
                 creator_id=user_id,
                 body_mime_type=u'text/html',
-                message_classifier=classifier
+                message_classifier=classifier,
+                creation_date=now,
+                modification_date=now,
```

Comment:
In my opinion, the creation date is the same as the date of modification for a new post (the same logic is used in the file systems for example).

"Semantically, this could be wrong, as querying for a modified post would now be completely skewed by this change.". Please see the `modified` attr. I think that the post is modified if the modification date is greater than the creation date.

However, I agree with you on adding a getter for the modification_date attribute (and not a new attribute).

Reply:
You shouldn't do it explicitly here. creation_date is defined like this on the `Content` class
`creation_date = Column(DateTime, nullable=False, default=datetime.utcnow)`

You added `modification_date = Column(DateTime)` to `AssemblPost. I think you should put 
`modification_date = Column(DateTime, nullable=False, default=datetime.utcnow)`
to the `Content` class instead (models/generic.py) instead, and remove this change here.

Reply:
Please see the latest changes :)

Reply:
it doesn't solve the problem. There will always be a difference between the dates.